test/AWS_Tests/src/S3_Spec.enso

```diff
@@ -5,50 +5,49 @@ import Standard.Base.Runtime.Ref.Ref
 from Standard.AWS import S3, AWS_Credential
 from Standard.AWS.Errors import AWS_SDK_Error, More_Records_Available, S3_Error, S3_Bucket_Not_Found
 
-from Standard.Test import Test, Test_Suite
-import Standard.Test.Extensions
+from Standard.Test_New import all
 
 import enso_dev.Base_Tests.Network.Enso_Cloud.Cloud_Tests_Setup.Cloud_Tests_Setup
 from enso_dev.Base_Tests.Network.Enso_Cloud.Cloud_Tests_Setup import with_retries
 
-spec =
+add_specs suite_builder =
     bucket_name = "enso-data-samples"
     not_a_bucket_name = "not_a_bucket_enso"
     object_name = "Bus_Stop_Benches.geojson"
     folder_name = "examples/"
     sub_folder_name = "examples/folder 1/"
     api_pending = if Environment.get "AWS_ACCESS_KEY_ID" . is_nothing then "No Access Key found." else Nothing
     cloud_setup = Cloud_Tests_Setup.prepare
+    can_list_buckets = S3.list_buckets.is_error.not
```

Comment:
I have only noticed this now, when I was merging these changes into my own PR.

This is definitely not what we are supposed to do - we are fetching S3 at the stage of gathering the tests. This is not good as gathering tests should not have side effects like network requests. It will also be slowed down quite a lot because of that.

I think we may need here a way to mark tests as pending _after_ they were run. Alternatively we need to change the way setup is performed so that tests to run are based only on environment variables and not on results of web requests.